lib/ansible/template/__init__.py

```diff
@@ -68,6 +68,14 @@
 
 JINJA2_OVERRIDE = '#jinja2:'
 
+USE_JINJA2_NATIVE = False
+if C.DEFAULT_JINJA2_NATIVE:
```

Comment:
I would the previous:
```from jinja2 import Environment```
into an 'else/pass' here to make the choice obvious vs the 'masking' of the previous import